packages/ui/src/composites/wui-list-select/index.ts

```diff
@@ -0,0 +1,63 @@
+import { html, LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import { globalStyles } from '../../utils/ThemeUtil'
+import '../../components/wui-text'
+import '../wui-wallet-image'
+import '../wui-all-wallets-image'
+import styles from './styles'
+import type { IWalletImage } from '../../utils/TypesUtil'
+
+@customElement('wui-list-select')
+export class WuiListSelect extends LitElement {
+  public static styles = [globalStyles, styles]
+
+  // -- state & properties ------------------------------------------- //
+  @property({ type: Array }) public walletImages?: IWalletImage[] = []
+
+  @property() public imageSrc? = ''
+
+  @property() public listName = ''
+
+  @property() public listStatus?: string
+
+  @property({ type: Boolean }) public showAllWallets = false
+
+  // -- render ------------------------------------------------------- //
+  public render() {
+    const getWalletComponent = () => {
+      if (this.showAllWallets && this.walletImages) {
+        return html`<wui-all-wallets-image
+          .walletImages=${this.walletImages}
+        ></wui-all-wallets-image>`
+      } else if (this.imageSrc) {
+        return html`<wui-wallet-image
+          size="sm"
+          src=${this.imageSrc}
+          alt=${this.listName}
+        ></wui-wallet-image>`
+      }
+
+      return undefined
+    }
+
+    const status =
+      this.listStatus &&
+      html` <wui-text variant="micro-700" color="fg-300">${this.listStatus}</wui-text> `
+
+    return html`
+      <button>
+        <div>
```

Comment:
Can we avoid using div here, by making `<wui-text>` for name (as these can be the longest) occupy all remaining available space in flex, thus pushing status label to the end?